src/mongo/grammar/mongoParser.ts

```diff
@@ -11,13 +11,17 @@
 
 import { ATN } from 'antlr4ts/atn/ATN';
 import { ATNDeserializer } from 'antlr4ts/atn/ATNDeserializer';
+// import { FailedPredicateException } from 'antlr4ts/FailedPredicateException';
 import { NotNull } from 'antlr4ts/Decorators';
 import { NoViableAltException } from 'antlr4ts/NoViableAltException';
 import { Override } from 'antlr4ts/Decorators';
 import { Parser } from 'antlr4ts/Parser';
 import { ParserRuleContext } from 'antlr4ts/ParserRuleContext';
 import { ParserATNSimulator } from 'antlr4ts/atn/ParserATNSimulator';
+// import { ParseTreeListener } from 'antlr4ts/tree/ParseTreeListener';
+// import { ParseTreeVisitor } from 'antlr4ts/tree/ParseTreeVisitor';
 import { RecognitionException } from 'antlr4ts/RecognitionException';
+// import { RuleContext } from 'antlr4ts/RuleContext';
```

Comment:
Remove commented lines if not used. 

I realize these may be auto-generated, I remember getting the green squiggly too.

Reply:
I'd rather leave them in, it makes it easier for the next person who's doing this to know that it's okay to manually remove them.